initial-config/static/globals.json

```diff
@@ -107,6 +113,18 @@
             "clientSecret": "google-client-secret",
             "clientID": "google-app-client-id",
             "callbackURL": "https://portal.yourcompany.com/callback/google"
+        },
+        "oauth2": {
+            "useOauth2": true,
```

Comment:
Not a good idea to have a `true` default here.

Reply:
Done